proxy/core/src/main/scala/io/cloudstate/proxy/EntityDiscoveryManager.scala

```diff
@@ -146,7 +146,7 @@ class EntityDiscoveryManager(config: EntityDiscoveryManager.Configuration)(
   private[this] final val clientSettings =
     GrpcClientSettings
       .connectToServiceAt(config.userFunctionInterface, config.userFunctionPort)
-      .withChannelBuilderOverrides(_.maxInboundMessageSize(config.maxInboundMessageSize))
+      .withChannelBuilderOverrides(_.maxInboundMessageSize(config.maxInboundMessageSize.toInt))
```

Comment:
This will potentially lead to problems if someone puts something outrageously large in the config, so I suggest that you add to the validation of the config parameters that the size does not exceed Int.MaxValue and give a sensible error message.

Reply:
So test the size in the code above and throw an Exception if it is too large?

Reply:
Yes, a ConfigurationException, see other properties for examples :)

Reply:
add check above